lib/libsys/open.2

```diff
@@ -195,6 +195,18 @@ error if file is not a directory
 .It Dv O_CLOEXEC
 automatically close file on
 .Xr execve 2
+.It Dv O_CLOFORK
+If set, the file descriptor returned will be closed in any child processses created with the
+.Fn fork 2
+family of functions.
+The file descriptor will remain open in the parent.
+This sets the
+.Dv FD_CLOFORK
+flag on the file descriptor.
+If not performed at open time, this can later be set with the
```

Comment:
Remove 'if not performed ...'.  The reference to F_SETFD is useful, so I propose to formulate it like 'The flag can be modified using F_SETFD ...'